.eslintrc.json

```diff
@@ -0,0 +1,31 @@
+{
+    "env": {
+        "node": true,
+        "es6": true
+    },
+    "parser": "espree",
```

Comment:
If we are going with the default, its fine to just take out the parser specifier.

Reply:
that's correct